### DISCUSSION TITLE
How to upgrade LibreSBC

### DISCUSSION THREAD

**Discussion by jj1682000** [hnimminh/libresbc#93] [General]
I have installed LibreSBC v0.5.9 Release.
so, I will upgrade LibreSBC from v0.5.9 to v0.6.0.
I only follow the installation below. right?
https://github.com/hnimminh/libresbc/wiki/Installation

Thanks a lot!

**Comment by hnimminh** [comment]
just rerun deployment with component **libre** , 
```ansible-playbook playbooks/deployment.yml -i inventories/production -l "<machine-name>" -t "libre"```

**Reply by jj1682000** [reply]
ok, Thanks!
